taichi_three/loader.py

```diff
@@ -62,6 +62,15 @@ def readobj(path, scale=None):
         else:
             assert False, len(indices)
 
+    if vi.__len__() == 0:
+        vi.append([0, 0, 0])
+    if vt.__len__() == 0:
+        vt.append([0, 0])
+    if vn.__len__() == 0:
+        vn.append([0, 0, 0])
+    if faces.__len__() == 0:
+        faces.append([0, 0, 0])
+
```

Comment:
Why these? `len(faces)` should also work.

Reply:
In model.py, but vx_n may never be None if use objloader.
```python
        if vi_n is None:
            vi_n = 1
        if vt_n is None:
            vt_n = 1
        if vn_n is None:
            vn_n = 1
```
When vn_n is zero, this part will throw an error:
```python
        if f_n is not None:
            self.faces = ti.Matrix.var(3, f_m, ti.i32, f_n)
        if vi_n is not None:
            self.vi = ti.Vector.var(3, ti.f32, vi_n)
        if vt_n is not None:
            self.vt = ti.Vector.var(2, ti.f32, vt_n)
        if vn_n is not None:
            self.vn = ti.Vector.var(3, ti.f32, vn_n)
        if tex_n is not None:
            self.tex = ti.Vector.var(3, ti.f32, tex_n)
```
I will write something better to fix this bug.